src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1944,6 +1944,31 @@ limitations under the License.
             <include cluster="Total Volatile Organic Compounds Concentration Measurement" client="false" server="true" clientLocked="true" serverLocked="false"></include>
         </clusters>
     </deviceType>
+    <deviceType>
+        <name>MA-dishwasher</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Dishwasher</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0075</deviceId>
+        <clusters lockOthers="true">
+            <include cluster="Identify" client="false" server="false" clientLocked="true" serverLocked="false">
+                <requireAttribute>IDENTIFY_TIME</requireAttribute>
+                <requireAttribute>IDENTIFY_TYPE</requireAttribute>
+                <requireCommand>Identify</requireCommand>
+            </include>
+            <include cluster="On/Off" client="false" server="false" clientLocked="true" serverLocked="false">
+                <requireAttribute>ON_OFF</requireAttribute>
+                <requireCommand>Off</requireCommand>
+                <requireCommand>On</requireCommand>
+                <requireCommand>Toggle</requireCommand>
```

Comment:
Again, not sure why these require* bits are here.  What do they do?

Reply:
Remove it.